### PR TITLE
[TE] - Fix true anomaly definition to include expected anomaly 

### PR DESCRIPTION
**Pull request by xiaohui-sun**
This is to fix the precision calculation in performance page.
https://thirdeye.corp.linkedin.com/app/#/manage/alerts/performance?duration=custom&endDate=1547366400000&startDate=1539586800000

### CODE REVIEW

**Comment by jihaozh** [comment]
probably hide LinkedIn internal URL in comments.

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java [hunk 1]
```diff
@@ -203,8 +203,8 @@ public void init(List<MergedAnomalyResultDTO> anomalies) {
       if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
           .equals(AnomalyFeedbackType.ANOMALY_NEW_TREND)) {
         isLabeledTrueAnomalyNewTrend = true;
-      } else if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
-          .equals(AnomalyFeedbackType.ANOMALY)) {
+      } else if (feedback != null && feedback.getFeedbackType() != null &&
+          (feedback.getFeedbackType().equals(AnomalyFeedbackType.ANOMALY) || feedback.getFeedbackType().equals(AnomalyFeedbackType.ANOMALY_EXPECTED))) {
```
Comment (apucher) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java]: isn't there an *isAnomaly()* helper?

Comment (xiaohui-sun) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java]: Put it into a helper function.

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java [hunk 2]
```diff
@@ -200,12 +202,14 @@ public void init(List<MergedAnomalyResultDTO> anomalies) {
...
       boolean isLabeledTrueAnomalyNewTrend = false;
-      if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
-          .equals(AnomalyFeedbackType.ANOMALY_NEW_TREND)) {
-        isLabeledTrueAnomalyNewTrend = true;
-      } else if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
-          .equals(AnomalyFeedbackType.ANOMALY)) {
-        isLabeledTrueAnomaly = true;
+      if (feedback != null && feedback.getFeedbackType() != null) {
+        AnomalyClassificationType anomalyClassificationType = ResourceUtils.getClassificationType(feedback.getFeedbackType());
+        if (anomalyClassificationType == AnomalyClassificationType.TRUE_POSITIVE) {
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java]: consider use feedback.getFeedbackType().isAnomaly() ? that one is in the feedback type label class, make it easier to manage the labels.

Comment (xiaohui-sun) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java]: Done.

**Pull request merged**